Makefile

```diff
@@ -25,19 +27,20 @@ db.migrate.status:
 
 fixtures.generate:
 	@echo "Generating fixtures ..."
-	@if [ ! -d fixtures/migrations ]; then mkdir -p fixtures/migrations; fi
+	# without file config.yaml (even empty), hasura migrate apply will fail
+	@if [ ! -d $(FIXTURES_MIGRATIONS_FOLDER) ]; then mkdir -p $(FIXTURES_MIGRATIONS_FOLDER); touch $(FIXTURES_FOLDER)/config.yaml; fi
```

Comment:
Interestingly, if you don't provide the `config.yaml` file, then the migration fails.

Reply:
Yes. I think hasura it's not for nothing that hasura is in beta ...